webauthn/helpers/cose.py

```diff
@@ -43,6 +45,7 @@ class COSEKTY(int, Enum):
     OKP = 1
     EC2 = 2
     RSA = 3
+    MLDSA = 7
```

Comment:
Might as well keep the naming here consistent with the way the algs above were defined:

```suggestion
    ML_DSA = 7
```